lib/node_modules/@stdlib/stats/base/dists/normal/logcdf/src/main.c

```diff
@@ -0,0 +1,65 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/stats/base/dists/normal/logcdf.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/erfc.h"
+#include "stdlib/math/base/special/erfcx.h"
+#include "stdlib/math/base/special/ln.h"
+#include "stdlib/math/base/special/log1p.h"
+#include "stdlib/math/base/special/abs2.h"
+#include "stdlib/constants/float64/ninf.h"
+
+#define INV_SQRT_TWO 0.7071067811865475  // 1/sqrt(2)
```

Comment:
Let's use `static const double` rather than `#define` for defining constants (offers type safety and scoped visibility of the contsant to the file, preventing any name clashes).